Adyen/Model/BalancePlatform/BalanceWebhookSettingAllOf.cs

```diff
@@ -0,0 +1,130 @@
+/*
+* Configuration API
+*
+*
+* The version of the OpenAPI document: 2
+*
+* NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+* https://openapi-generator.tech
+* Do not edit the class manually.
+*/
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Text.RegularExpressions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+using System.ComponentModel.DataAnnotations;
+using OpenAPIDateConverter = Adyen.ApiSerialization.OpenAPIDateConverter;
+
+namespace Adyen.Model.BalancePlatform
+{
+    /// <summary>
+    /// BalanceWebhookSettingAllOf
+    /// </summary>
+    [DataContract(Name = "BalanceWebhookSetting_allOf")]
+    public partial class BalanceWebhookSettingAllOf : IEquatable<BalanceWebhookSettingAllOf>, IValidatableObject
```

Comment:
It wouldn't consider this change in this release, `allOf` should be supported properly (avoiding `AllOf` postfix, use ineheritance).